api/src/main/java/io/kafbat/ui/model/rbac/permission/TopicAction.java

```diff
@@ -13,6 +13,8 @@ public enum TopicAction implements PermissibleAction {
   MESSAGES_READ(VIEW),
   MESSAGES_PRODUCE(VIEW),
   MESSAGES_DELETE(VIEW, EDIT),
+  ANALYSIS_VIEW(VIEW, MESSAGES_READ),
```

Comment:
I believe that both permissions here shouldn't have messages-related permissions here.
Being able to run analysis doesn't imply that the person should be able to read messages.
Besides that, the rest looks good now.

Reply:
done